### DISCUSSION TITLE
Create Chat panel component

### DISCUSSION THREAD

**Discussion by CarlosZiegler** [shadcn-ui/ui#736] [Ideas]
I as Developer that are using AI i think it will be great if we have some chat panel component and chat message text are or something like that. 

Like Vercel:

https://github.com/vercel-labs/ai-chatbot

**Comment by NikharPandya** [comment]
@CarlosZiegler that's already made with shadcn/ui

**Comment by jakobhoeg** [comment]
> I as Developer that are using AI i think it will be great if we have some chat panel component and chat message text are or something like that.
> 
> Like Vercel:
> 
> https://github.com/vercel-labs/ai-chatbot

You can also checkout [this](https://github.com/jakobhoeg/shadcn-chat) repository.

**Comment by MurzNN** [comment]
Found one more repo with Shadcn reusable chat components: https://github.com/miskibin/chat-components

**Comment by MurzNN** [comment]
And one more, even more popular: https://github.com/ibelick/prompt-kit
